packages/formation/sass/base/_b-variables.scss

```diff
@@ -61,49 +61,58 @@ $color-light-blue: #005ea2;
 $color-old-browser-background-start: #005ea2;
 $color-old-browser-background-end: #162e51;
 
-//===================================
-// Formation colors that override (or restate) USWDS values
-// Also see: https://standards.usa.gov/components/colors/
-//===================================
-
 $color-white:                #ffffff;
-$color-red:                  #e31c3d;
-$color-red-lightest:         #f9dede;
-$color-red-light:            #e59393;
-$color-red-dark:             #cd2026;
+$color-red:                  #d83933;
+$color-red-lightest:         #f8dfe2;
+$color-red-light:            #f2938c;
+$color-red-dark:             #b50909;
 
-$color-base:                 #212121;
+$color-base:                 #1b1b1b;
 
 $color-secondary:            $color-red;
 $color-secondary-lightest:   $color-red-lightest;
 $color-secondary-light:      $color-red-light;
 $color-secondary-dark:       $color-red-dark;
-$color-secondary-darkest:    #981b1e;
+$color-secondary-darkest:    #8b0a03;
 
-$color-gray:                 #5b616b;
-$color-gray-light:           #aeb0b5;
-$color-gray-lighter:         #d6d7d9;
-$color-gray-lightest:        #f1f1f1;
+$color-gray:                 #565c65;
+$color-gray-light:           #a9aeb1;
+$color-gray-light-alt:       #edeff0;
+$color-gray-lighter:         #dfe1e2;
+$color-gray-lightest:        #f0f0f0;
 $color-gray-medium:          #757575;
-$color-gray-dark:            #323a45;
-$color-gray-warm-light:      #e4e2e0;
-$color-gray-warm-dark:       #494440;
-$color-gray-cool-light:      #dce4ef;
+$color-gray-dark:            #3d4551;
+$color-gray-warm-light:      #e6e6e2;
+$color-gray-warm-dark:       #454540;
+$color-gray-cool-light:      #e1e7f1;
 
-$color-green:                #2e8540;
-$color-green-lightest:       #e7f4e4;
-$color-green-lighter:        #94bfa2;
-$color-green-light:          #4aa564;
+$color-green:                #008817;
+$color-green-lightest:       #ecf3ec;
+$color-green-lighter:        #b4d0b9;
+$color-green-light:          #5e9f69;
+$color-green-darker:         #154c21;
 
-$color-gold-lightest:        #fff1d2;
-$color-gold-lighter:         #fad980;
-$color-gold-light:           #f9c642;
+$color-gold-lightest:        #fef0c8;
+$color-gold-lighter:         #ffe396;
+$color-gold-light:           #face00;
+$color-gold:                 #ffbe2e;
+$color-gold-darker:          #936f38;
 
+$color-orange: #dd7533;
 
-$color-focus:                #3e94cf;
+$color-black:                #000;
 
+$color-va-accent:            #8a7237; // New gold "metal" acccent
+$color-feedback-warning-background:      #faf3d1; // Used for Disability benefits
+$color-inset-bg:             #e7f6f8;
+$color-gibill-accent:        #fef0c8; // TODO: Replace with $color-gold-lightest;
 $focus-outline:              2px solid $color-gold-light;
 
+$color-hub-family-member: $color-primary-darker;
+$color-hub-service-member: $color-primary-darker;
+
+$color-focus:                #3e94cf;
```

Comment:
How is $color-focus being used? Because #3e94cf would be closest to blue-40 (#4f97d1). I didn't realize this wasn't the same as vads-color-action-focus-on-light

Reply:
Looks like $color-focus is used for:

- accordion css class focus state: `.usa-accordion .usa-button-secondary:focus { box-shadow:  inset 0 0 0 2px $color-primary-darkest, 0 0 3px $color-focus, 0 0 7px $color-focus}`
- color-palette.scss: `.palette-container .palette-swatch .swatch.color-focus { background-color: $color-focus;}`